roofit/roofitcore/src/RooAbsOptTestStatistic.cxx

```diff
@@ -859,3 +862,27 @@ void RooAbsOptTestStatistic::setUpBinSampling() {
     _funcClone = newPdf;
   }
 }
+
+void RooAbsOptTestStatistic::setBatchMode(bool on) {
+
+  if(_batchMode == on) return;
+  _batchMode = on;
+
+  // if the cache optimization was never done to begin with, we also don't need
+  // to worry about reverting it
+  if(!_originallyCacheOptimized) return;
+
+  if(_batchMode) {
+    // this should do the opposite of `_funcClone->optimizeCacheMode(*_funcObsSet)` below
+    RooArgList serverList{};
+    _funcClone->treeNodeServerList(&serverList);
+    for (auto * arg : serverList) {
+        if(arg->operMode() != Auto) {
+            arg->setOperMode(Auto);
+        }
+    }
```

Comment:
Your editor is indenting with 3 spaces. Although legacy code is not always consistent in RF, the convention is 2 spaces and *never* tabs.

Reply:
Done, now it's 2 space.